OBSDOCS-917: Update installation docs (#73479)
openshift/openshift-docs: modules/distr-tracing-install-jaeger-operator.adoc

:_mod-docs-content-type: PROCEDURE
[id="distr-tracing-jaeger-operator-install_{context}"]
= Installing the {JaegerOperator} Operator

> You can install the {JaegerOperator} Operator through the link:https://operatorhub.io/[*OperatorHub*].

Reviewer: I suggest removing "You can" and just starting with "Install", This is a procedure for installing it, after all, so if the reader is here I'd think they want to install it.

Author: Thank you, I will keep this in mind when writing or updating other installation procedures. This particular product is deprecated, so I was reluctant to use the imperative mood here, and there is also a currently undocumented installation procedure using the CLI.

By default, the Operator is installed in the `openshift-operators` project.

.Prerequisites
* You have access to the {product-title} web console.
* You have access to the cluster as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.

> * If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerOperator} Operator.

Reviewer: I suggest using the existing attribute, {es-op}, for the OpenShift Elasticsearch Operator. This is especially important because there are licensing debates around Elasticsearch so who knows what decisions might be taken about it in the future?

I am also not sure the word "also" is needed here.

Author: Thank you. Good to know. I've applied your suggestion here manually.

.Procedure

. Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.

. Navigate to *Operators* -> *OperatorHub*.

. Search for the {JaegerOperator} Operator by entering *distributed tracing platform* in the search field.

. Select the *{JaegerOperator}* Operator, which is *provided by Red Hat*, to display information about the Operator.

. Click *Install*.

> . On the *Install Operator* page, select the *stable* for the *Update channel*. This will automatically update the Operator when new versions are released.

Reviewer: ```suggestion
. On the *Install Operator* page, select *stable* for *Update channel*. This setting automatically updates the Operator when new versions are released.
```

Reviewer: See https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs look for "tense" - the future tense in cause-and-effect sentences is discouraged.

Author: Thank you. I'm generally careful to avoid the future tense, but I didn't know about this IBM Style rule. I've thought about this sentence some more, and here is my end result:
```suggestion
. For the *Update channel* on the *Install Operator* page, select *stable* to automatically update the Operator when new versions are released.
```

//If you select a maintenance channel, for example, *Stable*, you will receive bug fixes and security patches for the length of the support cycle for that version.

. Accept the default *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and makes the Operator available to all projects in the cluster.

> . Accept the default *Automatic* approval strategy. By accepting the default, when a new version of this Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention. If you select *Manual* updates, when a newer version of an Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.

Reviewer: ```suggestion
. Accept the default *Automatic* approval strategy. If you accept the default, when a new version of this Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator without human intervention. If you select *Manual* updates, when a newer version of an Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.
```

Reviewer: "By accepting the default" looks like it's OLM that accepts the default.

Author: Indeed. Fixed. Thank you.

:_mod-docs-content-type: PROCEDURE
[id="distr-tracing-jaeger-operator-install_{context}"]
= Installing the {JaegerOperator} Operator

> You can install the {JaegerOperator} Operator through the link:https://operatorhub.io/[*OperatorHub*].

Reviewer 2: I won't block merge on this, but I would recommend making a follow-up PR to remove this `link:` here. OperatorHub.io is the upstream site for community Operators. Just saying "OperatorHub" (with no link) here is fine though / good enough, as that is what the related web console page on an actual OCP cluster is called too.

Author: OK. Thank you. Will do.

By default, the Operator is installed in the `openshift-operators` project.

.Prerequisites
* You have access to the {product-title} web console.
* You have access to the cluster as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
* If you require persistent storage, you must install the {es-op} before installing the {JaegerOperator} Operator.

.Procedure

. Log in to the {product-title} web console as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.

. Navigate to *Operators* -> *OperatorHub*.

. Search for the {JaegerOperator} Operator by entering *distributed tracing platform* in the search field.

> . Select the *{JaegerOperator}* Operator, which is *provided by Red Hat*, to display information about the Operator.

Reviewer 3: If `provided by Red Hat` is part of the Gui I'd provide a noun after it such as page or tab.

Author: Thank you. It's not part of the GUI.

. Click *Install*.

. For the *Update channel* on the *Install Operator* page, select *stable* to automatically update the Operator when new versions are released.
//If you select a maintenance channel, for example, *Stable*, you will receive bug fixes and security patches for the length of the support cycle for that version.

. Accept the default *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and makes the Operator available to all projects in the cluster.

. Accept the default *Automatic* approval strategy.
+
[NOTE]
====
If you accept this default, the Operator Lifecycle Manager (OLM) automatically upgrades the running instance of this Operator when a new version of the Operator becomes available.

> If you select *Manual* updates, the OLM creates an update request when a new version of the Operator becomes available. As a cluster administrator, you must then manually approve the update request to have the Operator updated to the new version. The *Manual* approval strategy requires a user with appropriate credentials to approve the Operator installation and subscription process.

Reviewer 3: ```suggestion
If you select *Manual* updates, the OLM creates an update request when a new version of the Operator becomes available. As a cluster administrator, you must then manually approve the update request to update the Operator to to the new version. The *Manual* approval strategy requires a user with appropriate credentials to approve the Operator installation and subscription process.
```

Reviewer 3: It might be helpful to be more specific as to what the `appropriate credentials` are.

Author: I've reworded the sentence in your second comment here. Thank you.

. Navigate to *Operators* -> *Installed Operators*.

> . On the *Installed Operators* page, select the `openshift-operators` project. Wait until you see that the {JaegerOperator} Operator shows a status of "Succeeded" before continuing.

Reviewer: I would use bold instead of quotes for Succeeded

Author: Good catch. I was only focused on updating the Operator names here.
These files where added when the engineering team had no technical writer assigned to their project, but I'm reluctant to go through it all because they plan to drop this product and all of its docs very soon.
